app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,98 @@
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+          <div class="navbar-nav">
+            <%= link_to "view top media", root_path, :class => "nav-item nav-link"%>
+            <li class="nav-item nav-link"> / </li>
+            <%= link_to "view all media", works_path, :class => "nav-item nav-link"%>
+            <li class="nav-item nav-link"> / </li>
+            <%= link_to "add a new work", new_work_path, :class => "nav-item nav-link"%>
+            <li class="nav-item nav-link"> / </li>
+            <%= link_to "view all users", users_path, :class => "nav-item nav-link"%>
+          </div>
+        </div>
+          <div class="col-1 d-flex justify-content-end align-items-center">
+            <% if session[:user_id] %>
+              <% @current_user = User.find_by(id: session[:user_id]) %>
+                <% if @current_user.nil? %>
+                  <%= button_to "Log In", login_path, method: :get, :class => "btn btn-custom-outline"  %>
+                <% else %>
+                  <%= button_to "You are currently logged in as #{@current_user.username}", user_path(@current_user), method: :get, :class => "btn btn-custom-outline mr-4" %>
+                  <%= button_to "Log Out", logout_path, method: :post, :class => "btn btn-custom-upvote" %>
+                <% end %>
+            <% else %>
+              <%= button_to "Log In", login_path, method: :get, :class => "btn btn-custom-outline"  %>
+            <% end %>
+          </div>
+      </nav>
+    </header>
+
+    <%# Content %>
+    <section class="content">
+      <div class="container-fluid p-5">
+        <section class="alert__container flash-msg__section">
+          <% flash.each do |name, message| %>
+            <% if name == "success" %>
+              <div class="alert alert-success light-text" role="alert">
+                  <strong class="<%= name %>">
+                    <%= message %>
+                  </strong>
+              </div>
+            <% else %>
+              <div class="alert alert-danger light-text" role="alert">
+                  <strong class="<%= name %>">
+                    <%= message %>
+                  </strong>
+              </div>
+            <% end %>
+          <% end %>
+        </section>
+        <%= yield %>
+      </div>
+    </section>
+    
+    <%# Footer %>
+    <footer class="footer text-center">
+      <div class="container mt-4">
+
+          <small>ALL IMAGES AND TEXT &copy; 2020 MEDIA RANKER </small> <br>
+          <small>Icons made by <a href="https://www.flaticon.com/authors/smalllikeart" title="smalllikeart">smalllikeart</a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a></small>
```

Comment:
Using `<small>` and `<br>` should generally be avoided if possible since they don't carry inherent semantic meaning.  (`<small>` was kind of retroactively given a meaning but should still be avoided when you can.)